src/main/java/wolox/training/repositories/UserRepository.java

```diff
@@ -0,0 +1,12 @@
+package wolox.training.repositories;
+
+import javax.validation.constraints.NotNull;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import wolox.training.models.User;
+
+public interface UserRepository extends CrudRepository<User, Long> {
+
+    @Query(value = "SELECT u FROM User u WHERE u.username = ?1")
```

Comment:
The @Query annotation is not required here, the CrudRepository handles it alone :) 
It is required when you want to perform more complex queries, where you need to specify some conditions.